### PR TITLE
Update to use ModuleLoader

### PR DESCRIPTION
**Pull request by Firesphere**
Updated to use the moduleloader from Framework to support vendor modules

### CODE REVIEW

File: docs/en/Installation.md [hunk 1]
```diff
@@ -37,4 +38,18 @@ SilverLeague\IDEAnnotator\DataObjectAnnotator:
...
+```yaml
+
+---
+Only:
+    environment: 'dev'
+---
+SilverLeague\IDEAnnotator\DataObjectAnnotator:
+    enabled_modules:
+      - mysite
+      - SilverLeague/IDEAnnotator
```
Comment (robbieaverill) [inline docs/en/Installation.md]: is this the module name or the namespace?

Comment (Firesphere) [reply docs/en/Installation.md]: Module name as specified in the composer.json. There is no way of trusting the namespace.

**Comment by robbieaverill** [comment]
Needs a rebase :-)

**Comment by Firesphere** [comment]
I'll get on to that tomorrow :) Soft, huggy, bed, sleepy time are currently slightly higher on the priority list :D

File: composer.json [hunk 2]
```diff
@@ -1,47 +1,52 @@
...
-        "scriptfusion/phpunit-immediate-exception-printer": "^2",
-        "phpunit/phpunit": "^5.7"
+    {
+      "name": "Simon 'Firesphere' Erkelens",
+      "email": "simonerkelens@silverstripe.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6.0",
+    "silverstripe/recipe-cms": "^1",
```
Comment (robbieaverill) [inline composer.json]: Generally requiring a recipe is a bad idea, it’s better to just require the cms module

Comment (Firesphere) [reply composer.json]: Good point. Let's see if that works with CI :)

**Pull request merged**